### PR TITLE
Mark doc as outdated

### PR DESCRIPTION
**Pull request by gwarf**
Mark doc as outdated

### CODE REVIEW

**Comment by gwarf** [comment]
The repo README and main page should also be updated.

File: source/index.rst [hunk 1]
```diff
@@ -2,6 +2,8 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+.. NOTE:: This documentation is obslote, please reach `<https://docs.egi.eu/users/cloud-compute/>`_.
```
Comment (enolfc) [inline source/index.rst]:
```suggestion
.. NOTE:: This documentation is obsolete, please reach `<https://docs.egi.eu/users/cloud-compute/>`_.
```

**Review by enolfc — CHANGES_REQUESTED** [review]
No doc commit without typo!

**Review by enolfc — APPROVED** [review]
It was good while it lasted 😄

**Pull request merged**